packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -42,27 +33,17 @@ abstract class PinCodeBlocEvents {
 
 /// A contract class containing all states of the PinCodeBloc.
 abstract class PinCodeBlocStates {
-  /// Returns true if the biometrics of the phone are enabled for usage from the app
-  Stream<bool> get areBiometricsEnabled;
-
-  /// Returns true is the pin code was verified by the API
-  ConnectableStream<bool> get isPinCodeVerified;
+  ///Indicating how much dots to present
+  Stream<int> get digitsCount;
 
-  /// Returns list of available biometrics if the biometrics for the app are
-  /// enabled, otherwise returns an empty list
-  ConnectableStream<List<BiometricsAuthType>> get availableBiometrics;
+  ///Indicating the biometrics button must be presented
+  ConnectableStream<bool> get showBiometricsButton;
 
-  /// Returns true if the user is authenticated with biometrics
-  ConnectableStream<bool> get isAuthenticatedWithBiometrics;
+  ///Emits when biometrics must be presented to the user
+  ConnectableStream<void> get requestBiometricsAuthentication;
```

Comment:
After a revision I've realised that we can remove this states. It is done by BiometricsService from the bloc.